### ISSUE TITLE
option to assign different resample methods to different requested gridded products

### ISSUE THREAD

**Issue by AmyHudson** [USDA-SCINet/rgeocdl#20]
If grabbing classification layer (e.g. crops) and continous layer (e.g. NDVI), would likely want 'nearest' for classification and 'bilinear' for classification.

**Comment by HeatherSavoy-USDA** [comment]
Good point! We have an open issue on the GeoCDL repo for this. The current plan is to keep 'nearest' the default for all variables, but let the user specify one method or two, one for categorical variables and one for continuous. We will add an error if a non-applicable method for categorical methods is requested. 

I'll leave this open until the GeoCDL issue is resolved and we have the documentation in rgeocdl updated for it.

**Comment by HeatherSavoy-USDA** [comment]
[This is partially implemented on GeoCDL](https://github.com/stuckyb/gcdl/issues/83)
